src/pages/_app.tsx

```diff
@@ -1,19 +1,45 @@
-import { FC } from 'react'
+import { FC, useContext } from 'react'
 import { AppProps } from 'next/app'
 import { SWRConfig } from 'swr'
-import GlobalStyle from '../components/GlobalStyle'
+import '../styles/fonts'
+import '../styles/globals.css'
+import { UserContext, UserContextProvider } from '../context/UserContext'
 
-const EmotionNextjsApp: FC<AppProps> = ({ Component, pageProps }) => (
-  <SWRConfig
-    value={{
-      fetcher<T>(url: string): Promise<T> {
-        return fetch(url).then((response) => response.json() as unknown as T)
-      },
-    }}
-  >
-    <GlobalStyle />
-    <Component {...pageProps} />
-  </SWRConfig>
-)
+const AppBar: FC = () => {
+  const { session } = useContext(UserContext)
+  return (
+    <div className="flex flex-row py-4 px-8 items-baseline justify-between">
+      <div className="text-3xl font-display font-extrabold">Coffee Chat</div>
```

Comment:
should we start using https://www.npmjs.com/package/clsx ? 
we found it pretty useful in our latest tailwind project.

Reply:
Looked at it. I don't think we need it yet. If we do, we can bring it in. But not yet given the current usage.